views/components/Graphs/BarGraph.jsx

```diff
@@ -39,9 +42,16 @@ const BarGraph = ({ graph }) => {
         {graph.data.map((data, idx) => (
           <VictoryBar 
             data={data} 
-            x='siteName' 
+            x='siteName'
             y='count' 
             key={idx}
+            labels={({ datum: { count, fieldLabel, siteName } }) => `Current: ${count} \n Target: ${(targetValuesMap[fieldLabel][siteName] && targetValuesMap[fieldLabel][siteName].value) ? targetValuesMap[fieldLabel][siteName].value : 'N/A'}`}
```

Comment:
Long Template string

Reply:
I think [Optional Chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) could help
```suggestion
            labels={({ datum: { count, fieldLabel, siteName } }) => `Current: ${count} \n Target: ${(targetValuesMap[fieldLabel][siteName]?.value || 'N/A'}`}
```

Reply:
Thanks.
[d28f2d6](https://github.com/AMP-SCZ/dpdash/pull/110/commits/d28f2d6b1be0f5c994babe2ddbf5581cbeed790c)